blosc/blosc2.h

```diff
@@ -1517,6 +1517,15 @@ BLOSC_EXPORT blosc2_frame* blosc2_frame_from_sframe(uint8_t *sframe, int64_t len
 BLOSC_EXPORT blosc2_schunk* blosc2_frame_to_schunk(blosc2_frame* frame, bool copy);
 
 
+/*********************************************************************
+  Directory utilities for needed for doing tests.
+*********************************************************************/
+
+/*
+ * Remove a directory and its files.
+ */
+BLOSC_EXPORT int remove_dir(const char *path);
```

Comment:
All the Blosc2 functions are prefixed with `blosc2_`, so rename this to `blosc2_remove_dir`.

Reply:
Fixed in 4607ab374e512f47bc7a64117e9af8907c18fc81